### ISSUE TITLE
null appended to accept-language header

### ISSUE THREAD

**Issue by ansar** [chao/RESTClient#146]
Hello,

When we tried to add the custom header Accept-Language:fr and make the rest request, We see that null is appended, so Accept-Language header become Accept-Language:null, fr. I saw there is a  code in file content/js/restclient.http.js line 46 set the accept-language as null. Could you please fix this.

Thanks and Regards,
Ansar Ahmed

**Comment by Finglas** [comment]
Been stung by this, very confusing at first. Shame because I've recommended this client quite heavily in the past, and know a fair few testers who rely on it.

@ansar is right, the offending line seems to be as stated, but in the latest version of the code I've tried removing this and running but no luck. The extension is also very different in terms of the UI so I think there are more changes that are in source, but not in the Firefox repo. I'm afraid I lack any real knowledge about Firefox extensions, all I can confirm is simply removing that line does not work for me (Windows 7 where this was tested).

**Comment by BojanKogoj** [comment]
This issue persists, any solution?

"Accept-Language": "null, de",

**Comment by gary-rowe** [comment]
Here's a link to my workaround that seems to be OK in production use:

https://github.com/bitcoin-solutions/multibit-website/blob/develop/src/main/java/org/multibit/site/servlets/SafeHeadersRequest.java

**Comment by BojanKogoj** [comment]
I don't think that is a good solution for me, I would have to update two Laravel packages which is a no-go.
